docs/src/advanced_features/frida/frida.md

```diff
@@ -0,0 +1,46 @@
+# Binary-only Fuzzing with Frida
+LibAFL supports binary-only fuzzing with Frida; the dynamic instrumentation tool.
+
+In this section, we'll talk about some of the components in fuzzing with `libafl_frida`.
+You can take a look at a working example in our `fuzzers/frida_libpng` folder.
+
+# Dependencies
+If you are on Linux, you'll need [libc++](https://libcxx.llvm.org/) for `libafl_frida` in addition to libafl's dependencies.
+They can be installed with
```

Comment:
Maybe note that this is the example for Linux, but macOS and win are supported too